index.html

```diff
@@ -126,7 +126,58 @@
 				</p>
 			</div> <!-- cd-faq-content -->
 		</li>
-	</ul> <!-- cd-faq-group -->
+	</ul> 
+        <ul id="frameworki" class="cd-faq-group">
+                <li class="cd-faq-title"><h2>Frameworki</h2></li>
+                <li>
+                        <a class="cd-faq-trigger" href="#0">Dlaczego hejtujemy Laravela?</a>
```

Comment:
Dobra - ponieważ ten wpis ma zasadniczo inny charakter, to proszę o jakieś bardziej szczegółowe informacje czego oczekujecie. Od koncepcji listy nie odejdę, bo pozwala ona krótko i zwięźle przedstawić zarzuty/wady/zalety. 
Ma być na bardo poważnie i w stylu manuala?